daemon/src/lib.rs

```diff
@@ -27,8 +27,8 @@ use tokio::sync::watch;
 use tokio_tasks::Tasks;
 use xtra::prelude::*;
 use xtra_bitmex_price_feed::QUOTE_INTERVAL_MINUTES;
-use xtra_libp2p::connection_monitor;
 use xtra_libp2p::dialer;
+use xtra_libp2p::endpoint_monitor;
```

Comment:
:wrench: Better if we could rename in the original commit that introduces this.

Reply:
it was done by different people at different times - prefer to keep the history.